assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -218,13 +226,26 @@ function SingleSelectionMovable( {
 			pinchable={ true }
 			keepRatio={ 'image' === selectedElement.type && isResizingFromCorner }
 			renderDirections={ ALL_HANDLES }
+			snappable={ true }
+			snapElement={ true }
+			snapHorizontal={ true }
+			snapVertical={ true }
+			snapCenter={ true }
+			horizontalGuidelines={ box.rotationAngle === 0 ?
+				[ 0, canvasHeight / 2, canvasHeight ] :
+				[ canvasHeight / 2 ] }
```

Comment:
I thought we saw that rotated shape and guidelines were working fairly well. No?